blueprints/mu-resource/index.js

```diff
@@ -66,9 +66,9 @@ module.exports = {
 
       var attr;
       if (/has-many/.test(dasherizedType)) {
-        var camelizedNamePlural = inflection.pluralize(camelizedName);
+        // var camelizedNamePlural = inflection.pluralize(camelizedName);
```

Comment:
if we don't use this code we should just remove it

Reply:
I removed it in the https://github.com/tenforce/ember-mu-application-generator/pull/4/commits/55826c68a6b863eb7865ff504b28dd484fe61adb commit